README.pt-BR.md

```diff
@@ -13,40 +13,43 @@
 [![build](https://img.shields.io/github/actions/workflow/status/hydralauncher/hydra/build.yml)](https://github.com/hydralauncher/hydra/actions)
 [![release](https://img.shields.io/github/package-json/v/hydralauncher/hydra)](https://github.com/hydralauncher/hydra/releases)
 
-[![be](https://img.shields.io/badge/lang-be-orange)](README.be.md)
+[![pt-BR](https://img.shields.io/badge/lang-pt--BR-green.svg)](README.pt-BR.md)
 [![en](https://img.shields.io/badge/lang-en-red.svg)](README.md)
-[![pl](https://img.shields.io/badge/lang-pl-white)](README.pl.md)
 [![ru](https://img.shields.io/badge/lang-ru-yellow.svg)](README.ru.md)
 [![uk-UA](https://img.shields.io/badge/lang-uk--UA-blue)](README.uk-UA.md)
+[![be](https://img.shields.io/badge/lang-be-orange)](README.be.md)
 [![es](https://img.shields.io/badge/lang-es-red)](README.es.md)
 [![fr](https://img.shields.io/badge/lang-fr-blue)](README.fr.md)
+[![ita](https://img.shields.io/badge/lang-it-red)](README.it.md)
 
 ![Hydra Catalogue](./docs/screenshot.png)
 
 </div>
 
 ## Índice
 
-- [Sobre](#about)
-- [Recursos](#features)
-- [Instalação](#installation)
-- [Contribuindo](#contributing)
-  - [Junte-se ao nosso Telegram](#join-our-telegram)
-  - [Fork e clone seu repositorio](#fork-and-clone-your-repository)
-  - [Como contribuir](#ways-you-can-contribute)
-  - [Estrutura do projeto](#project-structure)
-- [Compile a partir do código-fonte](#build-from-source)
-  - [Instale Node.js](#install-nodejs)
-  - [Instale Yarn](#install-yarn)
-  - [Instale Node Dependencies](#install-node-dependencies)
-  - [Instale Python 3.9](#install-python-39)
-  - [Instale Python Dependencies](#install-python-dependencies)
-- [variaveis de ambiente](#environment-variables)
-- [Rodando o programa](#running)
-- [Compilando](#build)
-  - [Compile o client bittorrent](#build-the-bittorrent-client)
-  - [Compile a aplicação Electron](#build-the-electron-application)
-- [Contribuidores](#contributors)
+- [Índice](#índice)
+- [ Sobre](#-sobre)
+- [ Recursos](#-recursos)
+- [ Instalação](#-instalação)
+- [ Contribuindo](#-contribuindo)
+  - [ Junte-se ao nosso Telegram](#-junte-se-ao-nosso-telegram)
+  - [ Fork e clone o seu repositório](#-fork-e-clone-o-seu-repositório)
+  - [ Formas de contribuir](#-formas-de-contribuir)
+  - [ Estrutura do Projeto](#-estrutura-do-projeto)
+- [ Compile a partir do código-fonte](#-compile-a-partir-do-código-fonte)
+  - [ Instale Node.js](#-instale-nodejs)
+  - [ Instale Yarn](#-instale-yarn)
+  - [ Instale Dependencias do Node](#-instale-dependencias-do-node)
+  - [ Instale Python 3.9](#-instale-python-39)
+  - [ Instale Python Dependencies](#-instale-python-dependencies)
+- [ Environment variables](#-environment-variables)
+- [ Running](#-running)
+- [ Build](#-build)
+  - [ Build the bittorrent client](#-build-the-bittorrent-client)
+  - [ Build the Electron application](#-build-the-electron-application)
+- [ Contributors](#-contributors)
+- [ Licença](#-licença)
```

Comment:
There is a space after the "[" in a lot of lines, and in other files as well. Is it intentional?

Reply:
i try to remove, but, may lint put the space again